Training/Easy/mime_type.rb

```diff
@@ -17,16 +17,20 @@ def init_associations
     associations
   end
 
+  def match_file(filename)
+    unknown = 'UNKNOWN'
+    splitted_filename = filename.split('.')
+    if splitted_filename.size > 1 && filename[-1] != '.'
+      result = associations.fetch(splitted_filename.last.downcase, nil)
+    end
+    result = unknown if result.nil?
+    result
+  end
+
   def start
     q.times do
```

Comment:
I understood. This was left from the example code added on the task. I will fix it.